File: translation_output/w_test/codellama/codenet/python/java/p00376.java
<s> [INST] <<SYS>>
You are an expert java programmer and assistant
<</SYS>>

Translate the following python code to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
20 30

Expected output:
10


```
a,b= map(int, input().split())
if a < b:
    print(b-a)
else:
    print(a-b)

```
[/INST]
```java
import java.util.Scanner;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        if (a < b) {
            System.out.println(b - a);
        } else {
            System.out.println(a - b);
        }
    }
}
```

Explanation:

* The `Scanner` class is used to read input from the user.
* The `nextInt()` method is used to read an integer from the user.
* The `split()` method is used to split the input string into two integers.
* The `map()` method is used to convert the input strings to integers.
* The `if` statement is used to check which number is smaller and print the difference between the two numbers.
* The `else` statement is used to print the difference between the two numbers if the first number is not smaller.

Note: The `import` statement is used to import the `Scanner` class from the `java.util` package.</s>

File: Results/DER/Translation/translation_output/w_test/mistral/codenet/java/python/p00376.py
<s> [INST] Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
20 30

Expected output:
10


```
import java.util.Scanner;

public class p00376
{
    public static void main(String[] args)
    {
        Scanner sc=new Scanner(System.in);
        int ans=0;

        int a=sc.nextInt();
        int b=sc.nextInt();
        
        ans=Math.abs(a-b);

        System.out.println(ans);
    }
}

```
[/INST]
```python
# Sample test case
a = int(input())
b = int(input())

# Expected output
ans = abs(a - b)

# Print the answer
print(ans)
```</s>
